### PR TITLE
timestamp version on generated foam-bin files

### PR DESCRIPTION
**Pull request by jlhughes**
Timestamp **version** on generated foam-bin files so localhost development forces load of new bin, rather than rely on forced reload from the browser.

### CODE REVIEW

File: tools/JSMaker.js [hunk 1]
```diff
@@ -30,8 +30,8 @@ exports.init = function() {
 
 
 exports.visitPOM = function(pom) {
+  if ( X.version ) version = X.version;
```
Comment (kgrgreer) [inline tools/JSMaker.js]: This should go in the init() method instead so you only do it once, not once for each POM.

File: tools/build.js [hunk 2]
```diff
@@ -280,18 +280,20 @@ task('Build web root directory for inclusion in JAR.', [], function jarWebroot()
 
   execSync(__dirname + `/pmake.js -makers=Webroot -pom=${pom()} -builddir=${BUILD_DIR}`, {stdio: 'inherit'});
 
-  function copy(foambin) {
-    if ( fs.existsSync('./' + foambin) ) {
-      copyFile('./' + foambin, webroot + '/' + foambin);
+  if ( PACKAGE || RUN_JAR ) {
```
Comment (kgrgreer) [inline tools/build.js:283]: Would this be easier if we just did an execSymc(`cp foam-bin-* ${webroot + '/' + foambin}`, {stdio: 'inherit'});
Maybe less portable though.
Do we clean up the old versions anywhere?

File: tools/JSMaker.js (hunk 1 above)
Comment (jlhughes) [reply tools/JSMaker.js]: moved.

File: tools/build.js (hunk 2 above)
Comment (jlhughes) [reply tools/build.js:283]: Moved cleanup to start of genJS call. 
And suggested copy works well.

**Pull request merged**